.github/workflows/build.yml

```
@@ -0,0 +1,77 @@
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        compiler: [ldc2, dmd]
+        arch: [x86_64, aarch64]
+        include:
+          - compiler: ldc2
+            dmd_binary: ldmd2
+          - compiler: dmd
+            dmd_binary: dmd
+        fail-fast: false
```

actionlint (1.7.12)

[error] expecting a single ${{...}} expression or array value for matrix variations, but found plain text node

(syntax-check)